lib/versions.js

```diff
@@ -48,14 +58,6 @@ module.exports = function(github, opts) {
         return tag;
     }
 
-    // Extract channel of version
-    function extractChannel(tag) {
-        var suffix = tag.split('-')[1];
-        if (!suffix) return 'stable';
-
-        return suffix.split('.')[0];
-    }
-
```

Comment:
Yeah, testing is the only reason why I moved it out.   It would be nicer to test the public function that calls this instead.   The only reason I didn't do that is time.  It would require mocking responses from Github.  (And there's no benefit for this function being in the closure).   Though that's probably a good thing to do in the end - and probably before submitting a PR into the main Nuts repo.

Hmm...  I'll see if I can knock out a quick mock and test things a better way.